examples/hybrid/sphere/baroclinic_wave_utilities.jl

```diff
@@ -473,14 +473,10 @@ end
 function output_writer(integrator)
     savefile = joinpath(
         output_dir,
-        test_name *
-        "_Day" *
-        string(Int(floor((integrator.t + tend) / 3600 / 24))) *
-        ".jld2",
+        "day" * string(Int(floor((integrator.t + tend) / 3600 / 24))) * ".jld2",
```

Comment:
Any reason why `test_name` is removed?

Reply:
same question from me

Reply:
The test_name global variable contains both the test file directory (e.g., sphere) and the test file name (e.g., baroclinic_wave_rhoe). I could have replaced test_name with the test file name, but that is already included in the output directory. For example, day5.jld2 will be stored in hybrid/sphere/output/baroclinic_wave_rhoe, so there is no need to call it baroclinic_wave_rhoe_day5.jld2. If you do want to call it that, then perhaps we should also add the test names to the plots; e.g., baroclinic_wave_rhoe_v.jld2?

Reply:
Ok, how about this. Vote on what you would most rather see:

1. hybrid/sphere/output/baroclinic_wave_rhoe/baroclinic_wave_rhoe_day5.jld2
2. hybrid/sphere/output/baroclinic_wave_rhoe/day5.jld2
3. hybrid/output/sphere/baroclinic_wave_rhoe/baroclinic_wave_rhoe_day5.jld2
4. hybrid/output/sphere/baroclinic_wave_rhoe/day5.jld2
5. hybrid/output/sphere/baroclinic_wave_rhoe_day5.jld2
6. hybrid/output/sphere_baroclinic_wave_rhoe_day5.jld2
7. hybrid/output/baroclinic_wave_rhoe_day5.jld2

Reply:
To clarify, the current PR will result in option 2.

Reply:
I see. I do not care much about the folder because it will be overwritten by the environment var set in the slurm script (at least that's how this is being done in the current main), this is because we want to save it to scratch or storage on central so that the simulation won't explode one's home did. But I do think we should have it inthe folder name and the file name.

Reply:
so 3

Reply:
Should I also add the test name to the generated plots? Then I could use option 5 for everything.

Reply:
Nevermind, option 5 might make the CI more convoluted. I'll just add the test file names everywhere, but stick with option 3.

Reply:
👍

Reply:
Yea. Let's stay with 3… and make sure if people specify 'OUTPUT_DIR' then it will be OUTPUT_DIR'/baroclinic_wave_rhoe_day5.jld2'